python/cudf/cudf/_libxx/copying.pyx

```diff
@@ -34,3 +41,25 @@ def gather(Table source_table, Column gather_map):
         column_names=source_table._column_names,
         index_names=source_table._index._column_names
     )
+
+
+def shift(Column input, int offset, Scalar fill_value=None):
```

Comment:
Since we're checking for isinstance below we can't make this a Scalar here:
```suggestion
def shift(Column input, int offset, object fill_value=None):
```